packages/tooling/fast-tooling/src/data-utilities/mapping.mdn-data.ts

```diff
@@ -1208,3 +1208,26 @@ export function mapCSSSyntaxes(mdnCSS: MDNCSS): CSSSyntaxDictionary {
             {}
         );
 }
+
+/**
+ * Converts an inline CSS string to a property dictionary data format
+ */
+export function mapCSSInlineStyleToCSSPropertyDictionary(
+    style: string
+): CSSPropertiesDictionary {
+    const cssPropertyDictionary = {};
+
+    if (style) {
+        style.split(";").forEach((cssDeclaration: string) => {
+            const cssPropertyAndValue = cssDeclaration.split(":");
```

Comment:
This might require a bit more nuance, but is probably good enough for now. The scenario I'm thinking of is where a property like `background-image` has a `url` that might include `https://`. Edge case for inline styles but worth noting.